header.go

```diff
@@ -746,7 +758,8 @@ func (h *ResponseHeader) CopyTo(dst *ResponseHeader) {
 	dst.noDefaultDate = h.noDefaultDate
 
 	dst.statusCode = h.statusCode
-	dst.statusLine = append(dst.statusLine, h.statusLine...)
+	dst.statusMessage = append(dst.statusMessage[:0], h.statusMessage...)
```

Comment:
```suggestion
	dst.statusMessage = append(dst.statusMessage, h.statusMessage...)
```
Sorry to nitpick but `dst.Reset()` is already called above which makes `dst.statusMessage` length 0. So the `[:0]` isn't needed.